internal/csi/driver/camanager.go

```diff
@@ -107,7 +107,7 @@ func (c *camanager) run(ctx context.Context, updateRetryPeriod time.Duration) {
 				// Retry updating the root CA files.
 				go func() {
 					select {
-					// Wait for 5 seconds before retrying.
+					// Wait before retrying.
```

Comment:
note: Removed the duration in the comment because it's a variable which could change.